packages/cards-editor-middleware/src/utilities.ts

```diff
@@ -0,0 +1,191 @@
+import { statSync, readdirSync, readFileSync } from 'fs';
```

Comment:
Please use `promises` for asynchronous file access.

Reply:
Done